components/layout/stacked-link.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { ExternalLinkIcon } from "@heroicons/react/solid";
+import clsx from "clsx";
+import Link from "next/link";
+
+interface StackedListProps {
+  links: [
+    {
+      title: string;
+      url: string;
+    },
+  ];
+}
+
+const LinkClasses = (i: number) => {
+  return [i > 0 ? "border-t" : undefined, "mx-2 py-4 flex justify-between"];
+};
+
+export default function StackedLink(list: StackedListProps) {
+  const { links } = list;
+  return (
```

Comment:
If you write the integration tests for the `Isoman` page correctly, they should have covered all these lines.